languages/csharp/exercises/concept/casting/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+
+public class LaminateMaker
+{
+    private bool securityAlert;
+    public LaminateMaker(bool securityAlert)
+    {
+        this.securityAlert = securityAlert;
+    }
+    public string GetDisplayName(TeamSupport support)
+    {
+        if (support is Staff staff)
+        {
+            string priorityPersonnelText = string.Empty;
+            if (securityAlert && staff.GetType() == typeof(Security))
```

Comment:
What if we would remove the `securityAlert` field and just have `if (staff.GetType() == typeof(Security))`?

Reply:
done